DedicatedServer/Scripts/Libs/Zrx/ModeLibs/FlagRush/UI/Modules/Header.Script.txt

```diff
@@ -7,50 +7,48 @@
-			<gauge pos="-30 0" z-index="1" size="8 3" id="mapscore-gauge-team1-background" color="FFF" halign="center" valign="center" style="ProgressBar" drawbg="0" ratio="1" grading="0.5"/>
-			<gauge pos="30 0" z-index="2" size="8 3" id="mapscore-gauge-team2" color="F00" valign="center" style="ProgressBar" halign="center" drawbg="0" ratio="0.5" grading="0.5" drawblockbg="0"/>
-			<gauge pos="30 0" z-index="1" size="8 3" id="mapscore-gauge-team2-background" color="FFF" halign="center" valign="center" style="ProgressBar" drawbg="0" grading="0.5"/>
+			<quad pos="-30 0" z-index="0" size="60 8" id="team1-mapscore-background" valign="center" halign="center" style="UICommon64_1" substyle="BgFrameTilted1" colorize="004" scale="0.5"/>
+			<quad pos="30 0" z-index="0" size="60 8" id="team2-mapscore-background" valign="center" halign="center" style="UICommon64_1" substyle="BgFrameTilted1" colorize="400" scale="0.5"/>
+			<label id="team1-mapscore" pos="-30 0" z-index="2" size="30 5" text=" " valign="center2" halign="center" textsize="1"/>
+			<label id="team2-mapscore" pos="30 0" z-index="2" size="30 5" text=" " valign="center2" halign="center" textsize="1"/>
 		</frame>
-
 		<frame id="main">
 			<frame id="team-stats">
 				<quad pos="-50 0" z-index="0" size="12 8" valign="top" halign="right" id="team1-score-background" style="UICommon64_1" substyle="BgFrame1" colorize="00f"/>
 				<quad pos="50 0" z-index="0" size="12 8" halign="left" id="team2-score-background" style="UICommon64_1" substyle="BgFrame1" colorize="f00"/>
-				<label pos="-30 -4.5" z-index="1" size="38 8" text="CarrierTeam Blue" halign="center" textsize="4" id="team1-name" textfont="GameFontRegular" valign="center2"/>
-				<label pos="30 -4.5" z-index="1" size="38 8" text="CarrierTeam Red" valign="center2" halign="center" textsize="4" id="team2-name" textfont="GameFontRegular"/>
-				<label pos="-56 -4.5" z-index="2" size="12 8" text="0" halign="center" valign="center2" textsize="5" id="team1-score" textfont="GameFontBlack" />
-				<label pos="56 -4.5" z-index="2" size="12 10" text="0" halign="center" valign="center2" textsize="5" id="team2-score" textfont="GameFontBlack"/>
+				<quad pos="-33 -10" z-index="-3" size="8 8" valign="center" id="team1-matchscore-background" style="UICommon64_1" substyle="BgFrame1" colorize="00f"  halign="center" rot="90"/>
+				<quad pos="33 -10" z-index="-3" size="8 8" id="team2-matchscore-background" style="UICommon64_1" substyle="BgFrame1" colorize="f00" halign="center" valign="center"/>
+				<label pos="-33 -11" z-index="0" size="8 8" text="0" halign="center" valign="center2" textfont="GameFontExtraBold" textsize="4" id="team1-matchscore"/>
+				<label pos="33 -11" z-index="0" size="8 8" text="0" halign="center" valign="center2" textfont="GameFontExtraBold" textsize="4" id="team2-matchscore"/>
```

Comment:
There's 2 numbers for each team without description now. The flag scores in a round and the map won by a team. For the round wins be have a label that specifically shows that it's round wins.

But since the "flag scored in a round" and "map wins" do not have such a label, it's not really obvious which one is what. With just one unlabeled number it's easier to guess that it's related to the objective of the gamemode, i.e scoring the flag. WIth 2 unlabeled numbers it's not that easy anymore.

This was always the problem with displaying the map wins. And I really don't like it the way this is now.